vaas-app/src/vaas/cluster/cluster.py

```diff
@@ -25,16 +26,32 @@ def make_parallel_loader(max_workers=settings.VAAS_LOADER_MAX_WORKERS,
     else:
         return ParallelLoader(max_workers)
 
-
 @app.task(bind=True, soft_time_limit=settings.CELERY_TASK_SOFT_TIME_LIMIT_SECONDS)
 def load_vcl_task(self, emmit_time, cluster_ids):
+    emmit_time_aware = timezone.make_aware(datetime.strptime(emmit_time, "%Y-%m-%dT%H:%M:%S.%fZ"),
+                                           timezone=timezone.utc)
+    if settings.STATSD_ENABLE:
+        total_processing_load_vcl_task_time = timezone.now() - emmit_time_aware
```

Comment:
I think it variable should be named not total_processing_load_vcl_task_time but queue_time_from_order_to_execute_task

Reply:
the name of variable has been changed according to your suggestion